packages/server/src/arpa_reporter/db/reporting-periods.js

```diff
@@ -42,6 +42,9 @@ async function getAllReportingPeriods(trns = knex, tenantId = useTenantId()) {
 /* getReportingPeriod() returns a record from the reporting_periods table.
   */
 async function getReportingPeriod(period_id = undefined, trns = knex, tenantId = useTenantId()) {
+    if (trns == null) {
+        trns = knex;
+    }
```

Comment:
Oh, that's a bit unfortunate!

Can we pass `undefined` somewhere instead of null so that this isn't necessary?